canvas_modules/common-canvas/__tests__/common-properties/controls/oneofselect-test.js

```diff
@@ -522,10 +522,29 @@ describe("oneofselect with custom value allowed works correctly", () => {
 		let dropdownWrapper = wrapper.find("div[data-id='properties-oneofselect-custom']");
 		const dropdownInput = dropdownWrapper.find("input");
 		dropdownInput.simulate("click");
+		// Enter 'e' in input to check [one, three] are filtered using shouldFilterItem
+		dropdownInput.simulate("change", { target: { value: "e" } });
```

Comment:
Hi @caritaou I have added this line because after adding shouldFilterItem dropdownList is having 0 items, is this fine ?

Reply:
Hi @srikant-ch5, in this case, shouldn't the list show all options since nothing was entered? I would expect this test to behave like this example:
![enum filter](https://github.com/user-attachments/assets/56419cd9-195e-4c24-b21b-2e62a1925eed)
If the test is wrong, can you help fix it using this example? This test is to verify the enumFilter works correctly

Reply:
Hi @caritaou yes after adding `shouldFilterItem` in `ComboBox` it is working as expected please refer below recording which is same as recording attached above.

https://github.com/user-attachments/assets/6f9f36f9-3f09-467e-951f-090e87bc16e2

As it is checking for `inputValue` in newly added function I have added`input.target.value=""` condition in test case.